admin-dev/themes/default/js/bundle/module/module.js

```diff
@@ -335,6 +334,10 @@ var AdminModuleController = function() {
     var body = $('body');
 
     body.on('change', this.bulkActionDropDownSelector, function() {
+      if (0 === $(self.getBulkCheckboxesCheckedSelector()).length) {
+          $.growl.warning({message: "You need to select at least one module to use the bulk action."});
```

Comment:
@AlexEven This is the typical messages currently not translated.

Another PR will follow for this task.